mobify/progressive-web-scaffold
# Add Spline

Add Spline-SCSS to the scaffold and apply the basic utility mixins to its utility classes

## Changes
- Add proper reference to Spline-SCSS' npm package and version
- Apply spline mixins to the scaffold utility classes

## How to test-drive this PR
- Run `npm install`
- Open a container like `home/container.js`
- Add a utility class, like `u-color-success`, to an element (with text) in the container
- Run `npm run dev`
- [Preview Merlin's Potions](https://preview.mobify.com/?url=http%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Confirm that the element's text is colored green
- Try adding some other classes from [the available options](https://github.com/mobify/progressive-web-scaffold/tree/add-spline/app/styles/utilities) to elements, confirming that they work as expected
